benchmarks/benchmark.py

```diff
@@ -126,7 +126,7 @@ def benchmark_suite(prepare: Callable[..., Callable], params_list: List[Dict],
 
   if FLAGS.export_dir:
     filename = _export_results(data_header, data, FLAGS.export_dir, name)
-    print("Wrote %s results to %s" % (name, filename))
+    print("Wrote {} results to {}".format(name, filename))
```

Comment:
Is there any option to make this use f-strings instead? That seems like a clearer choice here.

Reply:
This should be fixed now – it is simply because it can only do one "upgrade" at a time (%s -> {}, then next time pyupgrade is run {} -> f-string).